### ISSUE TITLE
Build error when using package:web v1.0.0

### ISSUE THREAD

**Issue by hnvn** [simolus3/moor#3142]
Step to reproduce:
- upgrade `web` package to v1.0.0
```
   web: ^1.0.0
```
- build project and got this error:
```
../../../../.pub-cache/hosted/pub.dev/drift-2.19.1+1/lib/src/web/wasm_setup.dart:111:60: Error: The argument type 'String' can't be assigned to the parameter type 'JSAny'.
          _DriftWorker.dedicated(web.Worker(driftWorkerUri.toString()));
```

**Comment by simolus3** [comment]
I have just published a version of `package:sqlite3` supporting the stable `web` version. I've adapted that in drift in 9f41a6ff3aad3be3229ab06bdad3215d992e4a26, but I probably won't be able to prepare a full drift release right away. Can you use the [development version](https://drift.simonbinder.eu/docs/internals/) in the meantime if you depend on that `web` version?

**Comment by simolus3** [comment]
Since others have asked about this as well, I've backported the fix and released it in drift 2.19.2.

**Comment by hnvn** [comment]
thank you so much
